server/controllers/userController.js

```diff
@@ -82,6 +82,31 @@ class UserController {
       signupMethod: user.signupMethod
     });
   }
+
+  static async emailSignin(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await models.Users.findOne({ where: { email } });
+      if (!user) return util.errorstatus(res, 404, 'User not found!');
+     
+      const result = await auth.comparePassword(password, user.password);
+
+      if (!result) return util.errorstatus(res, 403, 'Incorrect password!');
```

Comment:
I think a 401 error code is more appropriate here bro. Unauthorized as opposed to forbidden

Reply:
Ok. will change that.